contributors/triage/README.md

```diff
@@ -0,0 +1,13 @@
+# Kubernetes Issue Triage
```

Comment:
I like the round-robin for triage. Do we imagine an addition to the OWNERS files to specify a few people to triage per SIG?